generators/app/index.js

```diff
@@ -33,8 +32,8 @@ module.exports = class extends Generator {
         store: true,
         choices: [
           {
-            name: '0.12',
-            value: '12',
+            name: '0.13',
+            value: '13',
           },
           {
             name: '0.11',
```

Comment:
Why are we leaving version 11 in there? If we removed 12, can we remove 11 too and just force everything to be 13?